cfme/containers/provider/__init__.py

```diff
@@ -69,6 +69,11 @@ class default(Tab, DefaultEndpointForm, BeforeFillMixin):  # NOQA
         trusted_ca_certificates = TextInput('default_tls_ca_certs')
         api_port = Input('default_api_port')
 
+    @View.nested
+    class Virtualization(default):
```

Comment:
**REQUIRED**: Use `snake_case` for nested views together with `# NOQA` so linter does not complain.

Reply:
changed.